Makefile

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Mainflux
 # SPDX-License-Identifier: Apache-2.0
 
+MF_DOCKER_IMAGE_NAME_PREFIX ?= mainflux/
```

Comment:
Why not just `mainflux`? I would prefer that service is always prefixed with some name, then followed by `/`, so you can have `$(MF_DOCKER_IMAGE_NAME_PREFIX)/$(svc)`

It is more readable then concatenated like this.

Reply:
I thought it would be more flexible